content/en/continuous_integration/setup_pipelines/jenkins.md

```diff
@@ -172,7 +172,7 @@ Second, enable job log collection on the Datadog Plugin:
 
 1. In the web interface of your Jenkins instance, go to **Manage Jenkins > Configure System**.
 2. Go to the `Datadog Plugin` section, scrolling down the configuration screen.
-3. Select the `Datadog Agent` mode.
+3. Select the `Datadog Agent` mode. Data collection using Datadog API URL and API key is **not supported**.
```

Comment:
I have my doubts on this one because technically you can send logs to Datadog without an agent, but as you need to select the "Agent" mode to send CI Visibility data, this is a consequence. 
```suggestion
3. Select the `Datadog Agent` mode.
```

Reply:
Should we be more specific then?

```suggestion
3. Select the `Datadog Agent` mode. CI Visibility is **not supported** using Datadog API URL and API key. In order for log collection to work along CI Visibility, the Datadog API URL and API key should not be selected.
```

Reply:
You cannot select both "Datadog Agent" and "Datadog API URL & API Key" at the same time, so that's why I find it redundant.